flink-table/flink-sql-parser-hive/src/test/java/org/apache/flink/sql/parser/hive/FlinkHiveSqlParserImplTest.java

```diff
@@ -187,13 +187,13 @@ public void testConstraints() {
 		sql("create table tbl (x int,y timestamp not null,z string,primary key (x,z) disable novalidate rely)")
 				.ok("CREATE TABLE `TBL` (\n" +
 						"  `X`  INTEGER,\n" +
-						"  `Y`  TIMESTAMP(9) NOT NULL ENABLE NOVALIDATE RELY,\n" +
+						"  `Y`  TIMESTAMP NOT NULL ENABLE NOVALIDATE RELY,\n" +
 						"  `Z`  STRING,\n" +
 						"  PRIMARY KEY (`X`, `Z`) DISABLE NOVALIDATE RELY\n" +
 						")");
 		sql("create table tbl (x binary,y date,z string,constraint pk_cons primary key(x))")
 				.ok("CREATE TABLE `TBL` (\n" +
-						"  `X`  VARBINARY(2147483647),\n" +
+						"  `X`  BINARY,\n" +
 						"  `Y`  DATE,\n" +
```

Comment:
Does this mean we actually parse the type `BINARY` as `VARBINARY(2147483647)` ?

Reply:
Yes, it's the same as `BYTES` in Flink

Reply:
But Flink also has `BINARY` type, i think we should use that.

Reply:
No, Hive `BINARY` should be mapped to Flink `BYTES`, according to out type mappings:
https://ci.apache.org/projects/flink/flink-docs-release-1.10/dev/table/hive/hive_catalog.html#supported-types